device/celestica/x86_64-cel_seastone-r0/plugins/sfputil.py

```diff
@@ -41,7 +41,8 @@ def __init__(self):
 
         for x in range(self.PORT_START, self.PORT_END + 1):
             if self.port_start == 1:
-                self.port_to_eeprom_mapping[x] = eeprom_path.format((x - 1) + 26)
+                self.port_to_eeprom_mapping[
+                    x] = eeprom_path.format((x - 1) + 26)
```

Comment:
This line break looks like an accident to me. Can you fix it?

Reply:
Fixed in 5db212356dfce548f532d8398de43fed8dccf973